MuonAnalyser/plugins/MuonAnalyser.cc

```diff
@@ -838,6 +854,27 @@ void MuonAnalyser::fillBranches(TTree *tree, TLorentzVector tlv, edm::RefToBase<
       }
       
     }
+    
+    std::vector<double> me0tmvaValues;
+    if (b_muon_ME0dPhiBend != 100) { me0tmvaValues.push_back(fabs(b_muon_ME0dPhiBend)); }
```

Comment:
I tried without if and else first, but then bdt response didn't work well.. so I thought have to be removed initial values